lib/commands/web.js

```diff
@@ -112,9 +150,42 @@ extensions.getExtraNativeWebTapOffset = async function () {
   return offset;
 };
 
+async function tapWebElementNatively (driver, atomsElement) {
+  // try to get the text of the element, which will be accessible in the
+  // native context
+  try {
+    let text = await driver.executeAtom('get_text', [atomsElement]);
```

Comment:
i'm not sure this will work the way we expect. my recollection is that the get text atom gets the text of the element and all child elements, which doesn't correspond to what the text would be natively.

Reply:
Yes, but then we just fall back to computing the position, warts and all. At least the cases where this works (links, buttons, etc.) it is much faster and much more reliable.

Reply:
OK, I'm cool with that.